Gems/Atom/Feature/Common/Assets/Shaders/LightCulling/LightCullingHeatmap.azsl

```diff
@@ -6,16 +6,17 @@
  *
  */
 
-// The heatmap will change color as the light count increases up to TileCountMax, at which point and beyond it will be white.
+// The heatmap will change color as the light count increases up to TileCountMax
 static const int TileCountMax = 75;
+static const int OverflowDisplayNumber = 999;
 
 
-
-static const float3 BARELY_USED_COLOR = float3(0.05, 0.05, 0.20);
-static const float3 LIGHTLY_USED_COLOR = float3(0.05, 0.05, 0.60);
-static const float3 MODERATELY_USED_COLOR = float3(0.05, 0.60, 0.05);
-static const float3 HEAVILY_USED_COLOR = float3(1.00, 1.00, 0.05);
-static const float3 OVER_USED_COLOR = float3(1.00, 0.05, 0.05);
+static const float3 BarelyUsedColor = float3(0.05, 0.05, 0.20);
```

Comment:
small suggestion, perhaps add on the side the expected color for each to simplify matching it.
so - dark blue, blue, green, yellow, red

Reply:
I think thats a good idea. I will add it